src/main/java/com/lewisd/maven/lint/rules/AbstractReduntantVersionRule.java

```diff
@@ -47,20 +47,20 @@ protected void checkForRedundantVersions(final MavenProject mavenProject,
         }
 
         String version = resolveVersion(modelObject, resolvedModelObject);
-        String inheritedVersion = modelUtil.getVersion(tryResolveObject(inheritedObject));
+        String inheritedVersion = getModelUtil().getVersion(tryResolveObject(inheritedObject));

```

Comment:
The https://en.wikipedia.org/wiki/Liskov_substitution_principle talks about being able to substitute subclasses for their parent classes.  eg. passing a Rectangle to a method that expects a Shape.  I'm not sure how the Liskov substitution principle applies here.
The https://en.wikipedia.org/wiki/Uniform_access_principle does apply here, but I'm not dogmatic about it for utility fields like this since they're `final`.
I don't think leaving getters/setters leaves the object-oriented pattern, it just uses it differently.  It does expose the fact that they're fields, rather than abstracting that away behind methods, which would allow changing them from fields into something else, like singletons, but I feel like it's a worthwhile tradeoff for the clarity and brevity it results in.
```
modelUtil.getLocation(modelObject, "version");
```
is, I think, clearer, and certainly more brief than
```
getModelUtil().getLocation(modelObject, "version");
```
I think in this case, using a getter adds extra syntax without any benefit.